types/node-siege/node-siege-tests.ts

```diff
@@ -0,0 +1,13 @@
+import siege = require('node-siege');
+
+const attack = siege()
+    .args('-t5s -c 10 -b -f./urls.txt')
+    .on('stdout', (data) => {})
+    .on('stderr', (data) => {})
+    .on('error', (err) => {})
+    .on('exit', () => {
+        if (attack.running) {
+            throw new Error('Exit received but still marked as running');
+        }
+    })
+    .run();
```

Comment:
Can you add tests like

```ts
siege()
  .on(3000)
  .for(10000).times
  .get('/')
  .attack()
```

```ts
siege('/dir/name/app.js')
  .wait(1000)
  .get('/')
  .attack()
```

```ts
siege()
  .withCookie
  .post('/login', {user: pass}).for(1).times
  .get('/')
  .attack()
```

```
siege()
  .concurrent(100)
  .get('/')
  .attack()
```

Reply:
This is from a different package. These definitions are for this one: https://www.npmjs.com/package/node-siege